doc/conf.py

```diff
@@ -68,7 +72,7 @@ def __getattr__(cls, name):
 # -- General configuration ------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
-needs_sphinx = '1.6'
+needs_sphinx = "3.3"
```

Comment:
I assume this matches the other PL repo sphinx versions?

Reply:
This is from Ali's request. https://github.com/PennyLaneAI/pennylane-lightning/pull/253#discussion_r826223439

Reply:
This is the minimum Sphinx version in PL too; see https://github.com/PennyLaneAI/pennylane/blob/d29142589078a1508b2aa7bfe7906311e0d728b3/doc/conf.py#L27